pkg/minikube/cruntime/cri.go

```diff
@@ -30,16 +32,29 @@ func listCRIContainers(cr CommandRunner, filter string) ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
+	content = strings.TrimSpace(content)
```

Comment:
I'd just copy the result of `strings.Split` rather into another slice and skip empty strings. 
This will guard for a situation where there are empty lines in the output as well - in case they change the crictl output in the future, and it's equivalent to this check.

Reply:
Done.